dask_geopandas/core.py

```diff
@@ -0,0 +1,330 @@
+import numpy as np
+
+import dask.dataframe as dd
+from dask.dataframe.core import _emulate, map_partitions, elemwise
+from dask.utils import M, OperatorMethodMixin, derived_from, ignore_warning
+
+import geopandas
+from shapely.geometry.collection import GeometryCollection
+
+
+class _Frame(dd.core._Frame, OperatorMethodMixin):
+    """ Superclass for DataFrame and Series
+
+    Parameters
+    ----------
+    dsk : dict
+        The dask graph to compute this DataFrame
+    name : str
+        The key prefix that specifies which keys in the dask comprise this
+        particular DataFrame / Series
+    meta : geopandas.GeoDataFrame, geopandas.GeoSeries
+        An empty cudf object with names, dtypes, and indices matching the
+        expected output.
+    divisions : tuple of index values
+        Values along which we partition our blocks on the index
+    """
+
+    _partition_type = geopandas.base.GeoPandasBase
+
+    def __repr__(self):
+        s = "<dask_geopandas.%s | %d tasks | %d npartitions>"
+        return s % (type(self).__name__, len(self.dask), self.npartitions)
+
+    def to_dask_dataframe(self):
+        """Create a dask.dataframe object from a dask_geopandas object"""
+        return self.map_partitions(M.to_pandas)
```

Comment:
`map_partitions` calls the method on each partition, i.e. the underlying geopandas.GeoDataFrame/Series right? 
Because geopandas has no `to_pandas` method right now.

Reply:
Yeah I copied that from dask_cudf, but I don't understand how it could ever work. I am just going to delete it for now.

Reply:
This seems to have sneaked in again in a later commit?